bin/jquery.treeselect.js

```diff
@@ -1497,12 +1499,19 @@
       // Now declare the treeselect.
       var treeparams = params;
 
+      // Declare the callback function for selected node(s)
+      var singleSelectedNodeCallback = params.selected;
+      var selectingCompleteCallback = params.selectedAll;
+      
       // Reset the selected callback.
       treeparams.selected = (function(chosentree) {
 
         // Keep track of the selected nodes.
         var selectedNodes = {};
 
+        // Keep track of all selected nodes as an array for the callback function
+        var listOfSelectedNodes = [];        
```

Comment:
Thank you for your contribution.  However, I am not sure if I agree with this... this should be done by the library implementing this library and not this library itself.  We are using this library to load thousands of nodes and this will cause a large amount of memory to be consumed for that use case.  Can we just make this part of the library implementing this library and not bring it into the "core" library?  Thoughts?...